elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -701,6 +701,12 @@ def _cc_pipeline(
                                 container_op.add_pod_annotation(annotation.key, annotation.value)
                                 unique_annotations.append(annotation.key)
 
+                    # Force re-execution of the operation by setting staleness to zero days
+                    # https://www.kubeflow.org/docs/components/pipelines/overview/caching/#managing-caching-staleness
+                    if operation.disallow_cached_output:
+                        container_op.set_caching_options(enable_caching=False)
+                        container_op.execution_options.caching_strategy.max_cache_staleness = "P0D"
```

Comment:
"P0D" ...for a sec i thought this was a typo

Reply:
It does look that way but the format is https://www.ietf.org/rfc/rfc3339.txt